engines/grim/detection.cpp

```diff
@@ -153,6 +153,20 @@ static const GrimGameDescription gameDescriptions[] = {
 		},
 		GType_GRIM
 	},
+	{
+		// Grim Fandango Hebrew Fan translation (patched)
+		{
+			"grim",
+			"",
+			AD_ENTRY2s("LOCAL.LAB", "d22648d6787c2f8f0a789ee3ed0c08f7", 608,
```

Comment:
What's LOCAL.LAB? I don't have it. Can't find it anywhere in the [installation guide](https://www.facebook.com/groups/200491360554968/permalink/356883714915731) either.